src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -5,16 +5,33 @@
 import java.sql.SQLException;
 
 public class DatabaseManager {
+    private final String jdbcDriverClass;
+    private final String jdbcUrl;
     private Connection connection;
 
+    public DatabaseManager() {
+        jdbcDriverClass = "org.postgresql.Driver";
+        jdbcUrl = "jdbc:postgresql://localhost:5432/";
+    }
+
+    public DatabaseManager(String jdbcDriverClass, String jdbcUrl) {
```

Comment:
Who will use this constructor to create instance of DatabaseManager()? Should client know the details about jdbcDriverClass or jdbcUrl? In this case I think it's better to have jdbcDriverClass and jdbcUrl declared as a constants.

Reply:
It's just for testing of bad url and driverName)